src/kerberos_auth_server.py

```diff
@@ -113,10 +111,9 @@ def generate_session_key(self, client_id, server_id, nonce):
 
         key = client.get("passwordHash")
         bytes_key = str(key).encode()[32:]
-        # print(f"bytes: {bytes_key}, len: {len(bytes_key)}")
 
-        session_key = create_random_byte_key(16)
-        # client_key = encrypt_ng(bytes_key, {"encrypted_data": session_key, "nonce": nonce})
+        session_key = create_random_byte_key()
+        # TODO: if 'nonce_iv' not used, delete it
```

Comment:
you can't just delete it -> the function `encrypt_aes_ng` returns a tuple of {cipher iv)

Reply:
we can put "_" instead